doc/releases/release-notes-4.1.rst

```diff
@@ -586,11 +660,13 @@ New Drivers
    * :dtcompatible:`renesas,rz-scif-uart`
    * :dtcompatible:`silabs,eusart-uart`
    * :dtcompatible:`silabs,usart-uart`
+   * :dtcompatible:`ti,cc23x0-uart`
    * :dtcompatible:`wch,usart`
 
 * :abbr:`SPI (Serial Peripheral Interface)`
 
    * :dtcompatible:`ite,it8xxx2-spi`
+   * :dtcompatible:`nxp,lpspi`
```

Comment:
Not new, just renamed.

Reply:
Ya it's a challenge with several others actually actually (off the top of my head tdk has a few, wurthelektronik, prob others) -- need to figure out a way to flag these
Thanks!